src/site/antora/modules/ROOT/pages/manual/configuration.adoc

```diff
@@ -14,1856 +14,1019 @@
 
-[source,xml]
+. Files named `log4j2-test<contextName>.<extension>`
+. Files named `log4j2-test.<extension>`,
+. Files named `log4j2<contextName>.<extension>`
+. Files named `log4j2.<extension>`,
+. If no configuration file could be located a link:../javadoc/log4j-core/org/apache/logging/log4j/core/config/DefaultConfiguration[DefaultConfiguration] is used and a warning is printed by the status logger.
+The default configuration prints all messages less specific than
+xref:manual/systemproperties.adoc#log4j2.level[log4j2.level]
+to the console.
+
+[CAUTION]
+====
+The configuration files prefixed by `log4j2-test` should only be used on the test classpath.
+
+If multiple configuration files in the same category are found, Log4j uses a deterministic order to choose one of them (cf. link:../javadoc/log4j-core/org/apache/logging/log4j/core/config/Order[@Order]).
+Nevertheless, application developers are advised not to use multiple configuration files that differ only by the extension.
+
+Most libraries should not contain Log4j configuration files in their default location to allow applications to define their own.
+====
+
+The `<contextName>` and `<extension>` placeholders above have the following meaning
+
+<contextName>:: depends on the runtime environment in which Log4j runs:
+
+* for standalone Java SE application it is a random identifier,
+* for web applications it is derived from the application descriptor.
+See xref:manual/webapp.adoc#configuration[Log4j
+ Web application configuration] for more details.
+
+<extension>:: must be one of the file extensions assigned to a configuration file format:
++
+[cols="1,1"]
+|===
+| Configuration file format | Extension
+
+| XML
+| `xml`
+
+| JSON
+| `json` or `jsn`
+
+| YAML
+| `yaml` or `yml`
+
+|Java properties
+| `properties`
+|===
+
+[NOTE]
+====
+It is also possible to override the location of the configuration file using the
+xref:manual/systemproperties.adoc#log4j2.configurationFile[log4j2.configurationFile]
+configuration property.
+
+In this case Log4j will guess the configuration file format from the extension of the provided configuration file or will use the default configuration factory if the extension is unknown.
+See xref:manual/systemproperties.adoc#log4j2.configurationFile[log4j2.configurationFile] for details.
+====
+
+[id=configuration-syntax]
+=== [[ConfigurationSyntax]] Syntax
+
+All configuration files are parsed into a tree of link:../javadoc/log4j-core/org/apache/logging/log4j/core/config/Node[Node]s, each representing a different Log4j component.
+The root of the tree creates a link:../javadoc/log4j-core/org/apache/logging/log4j/core/config/Configuration[Configuration] object.
+
+A node is a fairly simple structure that represents a single Log4j plugin (cf. xref:plugin-reference.adoc[] for a complete list) such as an appender, layout or logger configuration.
```

Comment:
Sounds good to me.

I have lived in many Latin-centric countries, so these come naturally to me, but since the text is in English, there is no reason to use Latin.

BTW: Currently I see a tendency to use English expressions instead of Latin ones even in foreign languages.
The one I find most absurd is the usage of CV (pronounced /siˈvi/ instead of /tsefau/) in Polish.